### ISSUE TITLE
About  two softmax outputs

### ISSUE THREAD

**Issue by 321hallelujah** [sjenni/temporal-ssl#3]
Hi,
    When SSLtraining In your paper, you claim to use two softmax for the pseudo-task. However, in this code, it seems  you use tf.split to split the prediction（eg. 8 class for all, and 4 for skip ,the other 4 for transforms）, which lead to only one softmax. what is the difference about this, or just my misunderstanding. 
   Looking forward to your reply.

**Comment by sjenni** [comment]
Hi, 
Thanks for the question. The softmax is applied in the loss function, so in the end, two softmax functions are applied.
The code with splitting should be equivalent to two separate heads. Hope that clarifies the issue? :)
Cheers

**Comment by 321hallelujah** [comment]
thank you for your kindness.
because now I am intrested in Multi-task Self-Supervised Pre-traning. this great work can be regarded as a two-task Pre-traning.
so I'm curious whether the  splitting operation can apply to Multi-task classification. Is it a normal operation or something else. :)

**Comment by 321hallelujah** [comment]
thank you for your kindness.
because now I am intrested in Multi-task Self-Supervised Pre-training. this great work can be regarded as a two-task Pre-training.
so I'm curious whether the splitting operation can apply to Multi-task classification. Is it a normal operation in Self-Supervision or something else. :)

**Comment by 321hallelujah** [comment]
I got it!  the split variable is just fully_connected output of C3D.  So it can be split to any pseudo-task. The softmax is applied in every  loss function. Finally, the learned temporal-spatial representation is a concatenation style of two pseudo-task.
Is it right? Thanks
